src/main/docs/guide/ioc.adoc

```diff
@@ -1,6 +1,6 @@
 When most developers think of Inversion of Control (also known as Dependency Injection and referred to as such from this point onwards) the Spring Framework comes to mind.
 
-Micronaut takes inspiration from Spring, and in fact, the core developers of Micronaut are former SpringSource/Pivotal engineers now working for https://objectcomputing.com[OCI].
+Micronaut takes inspiration from Spring, and in fact, the core developers of Micronaut are former SpringSource/Pivotal engineers now working for https://objectcomputing.com[OCI] and https://labs.oracle.com[Oracle Labs].
```

Comment:
I think I would rather remove both companies there. Given the core developers may include other companies in the future it doesn't make sense to call out which specific ones have former springsource devs. Thoughts @jeffbrown ?

Reply:
Personally, I don't think mentioning that some of the current team used to work for Pivotal isn't terribly important.  The majority of the Micronaut core team did not work for SpringSource or Pivotal.

Reply:
I think this whole paragraph should just be removed and references to Spring